routes/aggrtimeout.go

```diff
@@ -6,34 +6,50 @@ import (
 	"strings"
 )
 
-func ConfigureAggrTimeout(route *InputRoute) *InputRoute {
+func parseTimeouts(v string) (int, error) {
```

Comment:
`parseTimeouts` returns an error for an empty input `v`.
Error: `strconv.Atoi: parsing "": invalid syntax`

Reply:
also it won't parse strings with a spaces `1 m`.

Reply:
fixed